README.md

```diff
@@ -2,15 +2,30 @@
 
 [![CircleCI](https://circleci.com/gh/twilio/twilio-video-room-monitor.js.svg?style=svg)](https://circleci.com/gh/twilio/twilio-video-room-monitor.js)
 
-_The Twilio Video Room Monitor is currently in beta. If you have feedback on how we can improve this tool going forward, we would love to hear from you. Feel free to provide this feedback by opening an Issue._
+*We want your feedback! Please feel free to open a [GitHub issue](https://github.com/twilio/twilio-video-room-monitor.js/issues) for suggested improvements or feature requests.*
 
 ## What is it
 
-This browser-based tool gives video app developers visibility and insights into the inner workings of any [Twilio Video JavaScript application](https://github.com/twilio/twilio-video.js). It displays real-time information that is gathered from the Video SDKs [Room object](https://media.twiliocdn.com/sdk/js/video/releases/2.14.0/docs/Room.html) and relevant browser APIs.
+A browser-based tool that provides [Twilio Video JavaScript](https://github.com/twilio/twilio-video.js) developers visibility into the inner workings of their video application. It displays near real-time information and metrics from the [Room object](https://media.twiliocdn.com/sdk/js/video/releases/2.14.0/docs/Room.html) and can be added to any Twilio Video JavaScript application with a few lines of code.
 
 ![Room Monitor gif](https://user-images.githubusercontent.com/40278237/127718088-8581c62d-13c1-4766-850d-14e4afd3ef08.gif)
 
-## Getting Started:
+The Twilio Video Room Monitor can be used as a tool during all phases of development or can be customized and provided to end-users as an in-call troubleshooting view. 
```

Comment:
When we say that it can be customized -- are there any methods on the monitor for customizing the view, or does this mean more that someone can fork this and develop a different version for their app's use case?

Reply:
@sarahcstringer I believe this is supposed to mean that users can clone this repo to make their own version of this app, and then customize the styling as they please by updating the CSS code. There currently isn't a way for users to update the appearance of the Room Monitor (but users can click and drag it around the page if they want!)